Highscores.csv

```diff
@@ -1,5 +1,10 @@
 Willemien,4,B
+Willemien,4,B
```

Comment:
ik vind het een beetje gek dat je dit bestaand ingecheckt hebt. Werkt je programma ook als het nog niet bestaat of moet je altijd ervoor zorgen dat er een bestand met de juiste naam voorhanden is? Maar een hele eer hoor :)